packages/tsx-editor/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "@uifabric/tsx-editor",
+  "version": "0.1.0",
+  "description": "Live tsx editor for Fabric website.",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "private": true,
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/OfficeDev/office-ui-fabric-react"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "npx just-scripts build",
+    "clean": "npx just-scripts clean",
+    "code-style": "npx just-scripts code-style",
+    "start": "npx just-scripts dev",
+    "start-test": "npx just-scripts jest-watch",
+    "update-snapshots": "npx just-scripts jest -u",
+    "just": "npx just-scripts"
+  },
+  "devDependencies": {
+    "@types/enzyme": "3.1.13",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/es6-promise": "0.0.32",
+    "@types/jest": "23.0.0",
+    "@types/react": "16.8.11",
+    "@types/react-dom": "16.8.4",
+    "@types/react-test-renderer": "^16.0.0",
+    "@types/webpack-env": "1.13.9",
+    "@uifabric/build": "^7.0.0",
+    "@uifabric/example-app-base": "^7.1.0",
+    "@uifabric/jest-serializer-merge-styles": "^7.0.2",
+    "@uifabric/prettier-rules": "^7.0.2",
+    "@uifabric/tslint-rules": "^7.0.2",
+    "enzyme": "~3.9.0",
+    "enzyme-adapter-react-16": "^1.2.0",
+    "es6-weak-map": "^2.0.2",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "react-test-renderer": "^16.3.0",
+    "webpack-bundle-analyzer": "^3.0.4"
+  },
+  "dependencies": {
+    "@microsoft/load-themed-styles": "^1.7.13",
+    "@uifabric/set-version": "^7.0.0",
+    "@uifabric/fluent-theme": "^7.0.2",
+    "monaco-editor": "^0.17.0",
+    "monaco-editor-webpack-plugin": "^1.7.0",
+    "office-ui-fabric-react": "^7.4.2",
```

To help fix the build error:
```suggestion
    "office-ui-fabric-react": "^7.4.3",
```